### PR TITLE
Changed the macOS ⌘ symbol to 'Ctrl+' on non-darwin platforms.

### PR DESCRIPTION
**Pull request by christianwico**
Revisions to the Tab Component to change the ⌘ into *Ctrl+* if the platform is not **darwin *(macOS/Mac OS X)***.

*(I'd recommend for someone with a Mac to first test if ⌘ instead appears if **upterm** is ran on a Mac. I "peasant-tested" this by switching the **if shorthand (TabComponent.tsx, line 48)** between **win32** and **darwin**. It seems to be working.*

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8896 @@
...
+          "requires": {
+            "color-convert": "1.9.0"
+          }
+        },
+        "chalk": {
+          "version": "2.0.1",
+          "resolved": "https://registry.npmjs.org/chalk/-/chalk-2.0.1.tgz",
+          "integrity": "sha512-Mp+FXEI+FrwY/XYV45b2YD3E8i3HwnEAoFcM0qlZzq/RZ9RwWitt2Y/c7cqRAz70U7hfekqx6qNYthuKFO6K0g==",
+          "dev": true,
+          "req
```
Comment (ForNeVeR) [inline package-lock.json]: I don't think you should introduce `package-lock.json` in an unrelevant PR. That should be the decision properly discussed with maintainer (probably in a new issue).

Comment (christianwico) [inline package-lock.json]: Heck. I accidentally used the *-a* upon commit. I didn't notice. Lemme fix it.

**Comment by christianwico** [comment]
Closing this PR in favor of #1141.

**Comment by ForNeVeR** [comment]
@christianwico just a small GitHub tip: you don't need to create a new PR if you want to replace contents of a previous one.

You could modify your commit with [`--amend`](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things) and [`--force`-push](https://stackoverflow.com/q/5509543/2684760) the new content to the same PR branch, and the PR will be replaced then.

**Comment by christianwico** [comment]
@ForNeVeR eheh. My apologies. I'm new to this GitHub culture. I'll remember next time.

**Pull request closed without merging**